installer/bundle/bundle_skel.sh

```diff
@@ -304,15 +304,15 @@ get_arch()
 
 compare_arch()
 {
-    #check if the user is trying to install the correct bundle (x64 vs. x86) 
+    #check if the user is trying to install the correct bundle (x64 vs. x86)
     echo "Checking host architecture ..."
-    AR=$(get_arch)
+    HOST_ARCH=$(get_arch)
     
     case $OMS_PKG in
-        *"$AR") 
+        *"$HOST_ARCH") 
             ;;
         *)         
-            echo "Cannot install $OMS_PKG on ${AR} platform"
+            echo "Cannot install $OMS_PKG on ${HOST_ARCH} bit platform"
```

Comment:
Since this value should still come out as "x64" or "x86", I think you can remove the "bit" from the echo.